Modules/Core/ImageAdaptors/test/itkComplexConjugateImageAdaptorTest.cxx

```diff
@@ -39,7 +39,7 @@ itkComplexConjugateImageAdaptorTest(int, char *[])
   itk::ImageRegionIterator<ImageType> iter(image, region);
   for (iter.GoToBegin(); !iter.IsAtEnd(); ++iter)
   {
-    auto            randMax = static_cast<float>(RAND_MAX);
+    auto            randMax = float{ RAND_MAX };
     const PixelType pixel(static_cast<float>(rand()) / randMax, static_cast<float>(rand()) / randMax);
```

Comment:
It does not look like anything is done with the value of the image before it is filled below. Maybe just use double, and then after division it can be reduced to float. I don't think it useful to working about system differences here.

Reply:
Thanks @blowekamp but for now I just reverted back to `static_cast<float>(RAND_MAX)`. Eventually I think we just shouldn't use the old `rand()` and `RAND_MAX` anymore. But that's for later  😺

Reply:
I don't think this code block is a productive test. I question the usefulness of making it beautiful portable modern C++. Time may be better spent writing a replacement test.

Reply:
> I don't think this code block is a productive test. I question the usefulness of making it beautiful portable modern C++. Time may be better spent writing a replacement test.

Well, I would prefer using GoogleTest, of course. But can you please be more specific on why you think the code block is not productive?

Reply:
It appears that the initial commit 75fd7073f39f948441845bc8902be60ce942f04c ("ENH: Added complex conjugate adaptor", Oct 26, 2011) already did so: first set a random number value for each pixel in the image, then fill the entire image buffer with `PixelType( 2.0, -3.7 )`:

https://github.com/InsightSoftwareConsortium/ITK/blob/75fd7073f39f948441845bc8902be60ce942f04c/Modules/Core/ImageAdaptors/test/itkComplexConjugateImageAdaptorTest.cxx#L40-L49

Is that why you don't think it's a productive test?

Reply:
Yes. The values are set and nothing is done with them. I suspect the initial test may have been with the random data, but it didn't work out.

Reply:
While I think the test _should_ work fine with arbitrary random input data, right?

@cquammen Hi Cory! Hope you're fine! We're just discussing your code from almost 14 years ago, I think 😃 I'm wondering, do you still remember why the `ComplexConjugateImageAdaptor` test first iteratively set the pixels of the image to random values, when it did `image->FillBuffer` directly afterwards anyway? What could have been the intention?

Reply:
Most likely an incomplete change in my testing methodology :-) Feel free to remove.

Reply:
Thanks @cquammen! Please check:
- pull request #5342